recipes/cpr/all/conanfile.py

```diff
@@ -44,7 +45,10 @@ def source(self):
         os.rename("cpr-{}".format(self.version), self._source_subfolder)
 
     def requirements(self):
-        self.requires("libcurl/7.67.0")
+        if Version(self.version) < "1.5.0":
+            self.requires("libcurl/7.67.0")
+        else:
+            self.requires("libcurl/7.69.1")
```

Comment:
```suggestion
        self.requires("libcurl/7.69.1")
```

I see no restriction for 7.69.1 declared in 1.4.0

Reply:
To my knowledge it is a feature of version [1.5.0 ](https://github.com/whoshuu/cpr/releases/tag/1.5.0)to use libcurl 7.69.1. Whether older version of cpr are fine with newer versions of curl I would need to ask.

Reply:
So please, ask. Keeping different versions is a problem when consuming multiple packages, it can result in conflicts.

Reply:
ping @COM8

Reply:
I'm currently the maintainer of `CPR` (https://github.com/whoshuu/cpr/tree/1.5.0#announcements)
`CPR` switched with [1.5.0](https://github.com/whoshuu/cpr/releases/tag/1.5.0) to libcurl `7.69.1`.
Older versions of `CPR` theoretically support libcurl `7.69.1`, but I definitely do not recommend this.

Reply:
Okay, let's keep the condition.